client/my-sites/stats/features/modules/stats-devices/stats-module-upgrade-overlay.tsx

```diff
@@ -51,7 +53,7 @@ const StatsModuleUpgradeOverlay: React.FC< StatsModuleUpgradeOverlayProps > = (
 	return (
 		// @ts-expect-error TODO: Refactor StatsListCard with TypeScript.
 		<StatsListCard
-			title="Devices"
+			title={ translate( 'Devices' ) }
```

Comment:
I didn't realize that Linear comments didn't sync to GH and created a [draft PR](https://github.com/Automattic/wp-calypso/pull/102432/files) for the same issue. The main difference is that `StatsModuleDevices` uses `import statsStrings from '../../../stats-strings';` to get the translations, so there is less duplication of strings. Could you update your PR to use `statsStrings`?